src/cluster.c

```diff
@@ -219,6 +223,32 @@ int auxShardIdPresent(clusterNode *n) {
     return strlen(n->shard_id);
 }
 
+int auxHumanNodenameSetter(clusterNode *n, void *value, int length) {
+    if (n && !strcmp(value, n->human_nodename)) {
+        return C_OK;
+    } else if (!n && (length == 0)) {
+        return C_OK;
+    }
+    if (n) {
+        n->human_nodename = sdscpy(n->human_nodename, value);
+    } else if (sdslen(n->human_nodename) != 0) {
+        sdsclear(n->human_nodename);
+    }
+	else
+	{
+		return C_ERR;
+	}
+	return C_OK;
```

Comment:
This spacing is trippy
```suggestion
    } else {
        return C_ERR;
    }
    return C_OK;
```

Reply:
sure, Removed the unnecessary spacings.